.env.example

```
@@ -0,0 +1,43 @@
+POSTGRES_PASSWORD=CHANGE_ME
+POSTGRES_DB=chat
+DATABASE_URL=postgres://chat:CHANGE_ME@postgres:5432/chat
+
+# Redis
+REDIS_URL=redis://redis:6379/0
+
+# API
+NODE_ENV=development
+PORT=3000
+SESSION_SECRET=CHANGE_ME_32_BYTES
+CSRF_SECRET=CHANGE_ME_32_BYTES
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CSRF_SECRET key should go before the NODE_ENV key

(UnorderedKey)